src/lib/OpenEXR/ImfStandardAttributes.h

```diff
@@ -140,24 +155,310 @@ IMF_STD_ATTRIBUTE_DEF (longitude, Longitude, float)
+// a linear crossfade between a Planckian blackbody locus and loci of
+// CIE D Series illuminants, the slope of the tint isotherm at the
+// exact points where the linear crossfade starts and ends can be 
+// indeterminate and an inverse mapping from chromaticity to a pair
+// of CCT and tint can be one-to-many.
+//
+// The most common use for the cameraTintSetting attribute is to feed its
+// value into a camera-vendor-provided application or API, along with
+// a cameraCCTSetting attribute value, to reproduce the color processing
+// done in-camera on set.
+//
+
+IMF_STD_ATTRIBUTE_DEF (cameraTintSetting, CameraTintSetting, float)
+
+//
+// cameraColorBalance -- chromaticity in CIE 1960 UCS coordinates
+// indicating a color the user of the camera would like the camera
+// to treat as neutral, and corresponding to a particular camera
+// configuration of make, model, camera firmware version, CCT setting
+// and tint setting.
+//
+// Note that this is not necessarily (or even probably) the same
+// chromaticity as that of the scene adopted white stored in an
+// adoptedNeutral attribute (if present).
+//
+// For example, if a physical digital cinema camera was configured with
+// a CCT of 3200K and a tint of -3 (in some camera vendor dependent unit),
+/// and the camera output had been processed such that the image containing
+// this attribute was encoded as per SMPTE ST 2065-4:2023, then 
+// the adoptedNeutral attribute would have the value corresponding to
+// the ACES neutral chromaticity, very near that of CIE Illuminant D60,
+// whereas the cameraColorBalance would have a chromaticity much, much
+// warmer than that of the adoptedNeutral attribute.
+// 
+
+IMF_STD_ATTRIBUTE_DEF (cameraColorBalance, CameraColorBalance, IMATH_NAMESPACE::V2f)
+
 //
 // isoSpeed -- the ISO speed of the film or image sensor
 // that was used to record the image
 //
 
 IMF_STD_ATTRIBUTE_DEF (isoSpeed, IsoSpeed, float)
 
+//
 // exposure -- exposure time, in seconds
+//
 
 IMF_STD_ATTRIBUTE_DEF (expTime, ExpTime, float)
 
 //
-// aperture -- the camera's lens aperture, in f-stops (focal length
-// of the lens divided by the diameter of the iris opening)
+// shutterAngle -- shutter angle, in degrees
+//
+
+IMF_STD_ATTRIBUTE_DEF (shutterAngle, ShutterAngle, float)
+
+//
+// captureRate -- capture rate, in frames per second, of the image
+// sequence to which the image belongs, represented as a rational
+// number
+// 
+// For variable frame rates, time-lapse photography, etc. the capture
+// rate r is calculated as
+//
+//   r = 1 / (tN - tNm1)
+//
+// where tn is the time, in seconds, of the center of frame N's 
+// exposure interval, and tNm1 is the time, in seconds, of the center
+// of frame N-1's exposure interval.
+//
+// Both the numerator and denominator of r must be strictly positive.
+//
+
+IMF_STD_ATTRIBUTE_DEF (captureRate, CaptureRate, Rational)
+
+//
+// timecodeRate -- the timecode rate associated with the playback of
```

Comment:
I understand why the capture rate is encoded. Why is a timecode rate encoded? In other words, what on-set reality is being recorded here? I'm trying to imagine how I might set up a camera capture sequence that might generate a capture rate and a timecode rate that differ. The only thing I can think of is a stop motion sequence, where perhaps I'm making 2 second exposures intended for 24fps playback. But that isn't a physical condition on the stage, the 24fps playback is an editorial choice, wouldn't that time code rate be imposed at the time that an image sequence is conformed into a media clip (or essence, as some vendors would have it)?